scripts/open-coredump.sh

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+#
+# Requirements:
+# * eu-unstrip
+# * curl
+# * jq
+# * git
+
+set -e
+
+function print_usage {
+cat << EOF
+Usage: ./open-coredump.sh [options] COREFILE
+
+Script for "one-click" opening of coredumps.
+
+It extracts the build-id from the coredump, retrieves metadata for that
+build, downloads the binary package, the source code and finally
+launches the dbuild container, with everything ready to load the
+coredump.
+
+The script is idempotent: running it after the prepartory steps will
+re-use what is already donwloaded. It is *strongly* recommended to run
+this from an empty directory, with nothing but the core-file present.
+
+Options:
+
+--help,-h
+    Print this help message and exit.
+
+--quiet,-q
+    Only print errors (if any).
+
+--progress
+    Print progress information when downloading the package and cloning
+    the git repo.
+
+--artifact-dir,-d ARTIFACT_DIR
+    Directory where the scrip will store all downloaded artifacts. If
+    the directory doesn't exist, it will be created.
+    It is strongly recommended that this is an empty directory, not used
+    for anything else, to avoid interference with the script.
+    To reset the state of the script, just delete this directory.
+    This can be the same as the directory the coredump is already in, or
+    some other path.
+    By default, a directory called open-coredump.sh.dir will be created in
+    the script's working directory.
+    Can be also provided via env variable ARTIFACT_DIR.
+    If both are provided command line has precedence.
+
+--scylla-s3-reloc-server-url SCYLLA_S3_RELOC_SERVER_URL
+    The URL of the s3 reloc server to connect to. Needed to fetch the
+    build info based on the build-id.
+    Defaults to http://backtrace.scylladb.com.
```

Comment:
It would be nice to put the default: `http://backtrace.scylladb.com` in a variable so it can be quoted here and below, when used.

nit: delete the terminating period, it's confusing a bit...

Reply:
Done in v4.